src/create_atoms.cpp

```diff
@@ -1050,7 +1051,12 @@ int CreateAtoms::add_quasirandom(const double vert[3][3], tagint molid)
     if ((point[0] >= sublo[0]) && (point[0] < subhi[0]) && (point[1] >= sublo[1]) &&
         (point[1] < subhi[1]) && (point[2] >= sublo[2]) && (point[2] < subhi[2])) {
 
-      atom->avec->create_atom(ntype, point);
+      if (mode == ATOM) atom->avec->create_atom(ntype, point);
```

Comment:
Using the "mesh" option with molecule files makes no sense. These changes need to be reverted.